distributions/JavaScript/INSTRUCTIONS

```diff
@@ -0,0 +1,27 @@
+1. Install Enscriptem
```

Comment:
Is this a duplication of `distributions/JavaScript/JS_INSTALL`. Please keep only one instruction (i..e remove old one if not necessary)

Reply:
Yes, It's a rewrite of the previous instruction, I will keep only one of them now.

Reply:
Good !